### DISCUSSION TITLE
Is SolidStart v1 using a fork of Nitro?

### DISCUSSION THREAD

**Discussion by quentindutot** [solidjs/solid-start#1443] [Q&A]
I found this Nitro discussion : [Who is using Nitro?](https://github.com/unjs/nitro/discussions/1015)

It has the following warning:
> Warning
> Sadly Solid (vinxi) decided to diverge from Nitro and fork it partially. Until further notice, we don't support any of it. Use at your own risk, and use their support channels if faced with any issues. Sorry for this.

There was a lot of changes during the betas and I wonder if this is still the case with the coming v1 release?

**Comment by ryansolid** [comment]
It is in a sense. We wrap it rather than say fork it. Nitro has certain features currently that are locked to file-based conventions which makes things difficult to have as much control over as we'd like. There is a server folder that offers some Nitro features directly, like API routes and others but they have their own conventions and we didn't want to introduce different ones from the main Start experience. For us the server and the client aren't meant to be far apart.

I anticipate there be ways to work around this in the future but since Vinxi aims to be a config wrapper similar to Vite itself it didn't make sense to adopt this. Fortunately Nitro is still quite usable in the way we are looking for so I'm very excited to be able to leverage it.
